### PR TITLE
feat(BIND): Adding support for USD in Argentina, and design updates for ARS layout.

### PR DESCRIPTION
**Pull request by lsena-bc**
## Description
Adding support to Argentinian users to deposit and withdraw USD by bank transfer. And minor design updates in the ARS flow (deposit and withdraw).

### CODE REVIEW

File: packages/blockchain-wallet-v4-frontend/src/modals/Brokerage/Banks/Deposit/WireInstructions/CurrencyLayouts/LayoutUsdArs/index.tsx [hunk 1]
```diff
@@ -0,0 +1,255 @@
...
+              <DisplayTitle>
+                <FormattedMessage
+                  id='modals.simplebuy.deposit.processing_time'
+                  defaultMessage='Processing Time'
+                />
+              </DisplayTitle>
+              <DisplaySubTitle>
+                <FormattedMessage
+                  id='modals.simplebuy.deposit.processing_time.info.usdars1'
+                  defaultMessage='Funds will be credited to your {currency} wallet as soon as we receive them. Funds are generally available within 3 business days. We will send you a mail with the confirmation.'
```
Comment (schnogz) [inline packages/blockchain-wallet-v4-frontend/src/modals/Brokerage/Banks/Deposit/WireInstructions/CurrencyLayouts/LayoutUsdArs/index.tsx:239]: `We will send you a mail with the confirmation.` should be `We will send you an email with the confirmation.`

File: packages/blockchain-wallet-v4-frontend/src/modals/Brokerage/Banks/Deposit/WireInstructions/CurrencyLayouts/default/index.tsx [hunk 2]
```diff
@@ -0,0 +1,472 @@
...
+                defaultMessage='Recipient'
+              />
+            </Title>
+            <Value data-e2e='sbRecipientName'>{recipientName}</Value>
+          </div>
+          <Copy>
+            <CopyClipboardButton textToCopy={recipientName} />
+          </Copy>
+        </RowCopy>
+        {['USD', 'EUR'].includes(props.account.currency) && (
```
Comment (schnogz) [inline packages/blockchain-wallet-v4-frontend/src/modals/Brokerage/Banks/Deposit/WireInstructions/CurrencyLayouts/default/index.tsx:204]: can you use fiat enum here?

File: packages/blockchain-wallet-v4-frontend/src/modals/Brokerage/Banks/Deposit/WireInstructions/CurrencyLayouts/default/index.tsx [hunk 3]
```diff
@@ -0,0 +1,472 @@
...
+                />
+              </Text>
+            </LegalWrapper>
+          )}
+        </BottomInfoContainer>
+      </Bottom>
+    </div>
+  )
+}
+
```
Comment (schnogz) [inline packages/blockchain-wallet-v4-frontend/src/modals/Brokerage/Banks/Deposit/WireInstructions/CurrencyLayouts/default/index.tsx:469]: tons of hardcoded EUR, GBP, USD strings.  can we replace with the fiat enums being used?

File: packages/blockchain-wallet-v4-frontend/src/modals/Brokerage/Banks/Deposit/WireInstructions/template.success.tsx [hunk 4]
```diff
@@ -1,281 +1,31 @@
...
-`
 
 const Success: React.FC<Props> = (props) => {
-  const [transferType, setTransferType] = useState(TransferType.DOMESTIC)
+  const userCountryCode = props.userData?.address?.country || 'default'
+
   const currencyLayouts = {
-    ARS: <LayoutArs {...props} />,
+    ARS: <LayoutUsdArs {...props} />,
+    USD: userCountryCode !== 'AR' ? <LayoutDefault {...props} /> : <LayoutUsdArs {...props} />,
```
Comment (schnogz) [inline packages/blockchain-wallet-v4-frontend/src/modals/Brokerage/Banks/Deposit/WireInstructions/template.success.tsx:20]: this doesnt make much sense?  why need this if default is the same as USD? also why checking country code here for the same layout as ARS?

File: packages/blockchain-wallet-v4-frontend/src/modals/Brokerage/Banks/Withdraw/WithdrawalDetails/index.tsx [hunk 5]
```diff
@@ -37,46 +38,77 @@ const SuccessIcon = styled(Icon)`
...
+              values={{
+                currency: fiatCurrency
+              }}
+            />
+          )}
+        </SubTitle>
+        <Button
+          fullwidth
+          height='48px'
+          data-e2e='withdrawReload'
```
Comment (schnogz) [inline packages/blockchain-wallet-v4-frontend/src/modals/Brokerage/Banks/Withdraw/WithdrawalDetails/index.tsx:96]: seems incorrect

**Review by schnogz — CHANGES_REQUESTED** [review]

File: packages/blockchain-wallet-v4-frontend/src/modals/Brokerage/Banks/Deposit/WireInstructions/CurrencyLayouts/LayoutUsdArs/index.tsx [hunk 6]
```diff
@@ -0,0 +1,255 @@
...
+import { DisplayIcon, DisplaySubTitle, DisplayTitle } from 'components/BuySell'
+import CopyClipboardButton from 'components/Clipboard/CopyClipboardButton'
+import { ExpansionPanel } from 'components/ExpansionPanel'
+import { FlyoutWrapper, Row, Title, Value } from 'components/Flyout'
+import { BankDWStepType } from 'data/types'
+
+import { Props as OwnProps, SuccessStateType } from '../..'
+
+const RowCopy = styled(Row)`
+  display: flex;
```
Comment (julianromero-bc) [inline packages/blockchain-wallet-v4-frontend/src/modals/Brokerage/Banks/Deposit/WireInstructions/CurrencyLayouts/LayoutUsdArs/index.tsx:17]: You could use Flex component from component package. You will find examples like
`<Flex flexDirection='row' justifyContent='space-between'>`

**Pull request merged**